docs/Статистика.md

```
@@ -0,0 +1,77 @@
+# Статистика
+
+Для детального анализа производительности предусмотрено добавление специализированных статистических показателей.
+
+## Min и Max
+
+#### Аннотации
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)